src/style.css

```
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+}
+
+.stars__star {
+  background: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(n) {
+  background: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
```

```suggestion
  background: url(./images/star-active.svg);
```
you already added these styles to `stars__star`